src/components/homePage/About.vue

```diff
@@ -31,68 +20,33 @@
 	margin: auto;
 	color: black;
 }
-@media screen and (max-width:1250px) {
+.about_section{
+	display: flex;
+	flex-direction: row;
+}
+.about_image{
+	width: 80vh;
+	height: auto;
+	margin-right: 10vh;
```

Comment:
Instead of adding a margin to image, please add the following line to the parent flexbox
`justify-content: space-between`

This will spread the elements to the edges (which is what we want!), and removes magic constants from the code

Reply:
So I tried using what you suggested..but doesn't seem like it's doing any effect. 

![spaceBw](https://user-images.githubusercontent.com/78088990/120039507-06526580-bfd3-11eb-8235-e42b2c362bfb.JPG)

this is the code of parent flexbox: (any suggestions?)
.about_section{
	display: flex;
	justify-content: space-between;
	flex-direction: row;
	color: black;
}

Reply:
Right, then the margin is okay :)